NEWS

```diff
@@ -1,3 +1,9 @@
+BoutrosLab.plotting.general 7.0.3 2022-03-15
+
+BUG
+* Update .Rbuildignore to match lab R standards
```

Comment:
Not sure if updating `.Rbuildignore` should be under `BUG`?

Reply:
> Not sure if updating `.Rbuildignore` should be under `BUG`?

Maybe "update" instead? I was unsure about this one. I didn't go with update initially, as it's not a new feature or anything.

Reply:
Update to me sounds better than bug but your call. Since the first word is "update" it seems like it makes sense :)

Reply:
Hahah, Good point! Update it is.